Jint/Native/Function/ScriptFunctionInstance.cs

```diff
@@ -138,7 +138,8 @@ ObjectInstance IConstructor.Construct(JsValue[] arguments, JsValue newTarget)
             {
                 try
                 {
-                    var result = OrdinaryCallEvaluateBody(_engine._activeEvaluationContext, arguments, calleeContext);
+                    var context = _engine._activeEvaluationContext ?? new EvaluationContext(_engine);
+                    var result = OrdinaryCallEvaluateBody(context, arguments, calleeContext);
```

Comment:
I know this change is required (because `_activeEvaluationContext` is used this way everywhere else). My app was crashing because of this, but I could not write a test which reproduces the issue in isolation.